OpenRA.Mods.Common/Traits/ProductionParadrop.cs

```diff
@@ -53,9 +53,8 @@ public override bool Produce(Actor self, ActorInfo producee, string factionVaria
 
 			// Start a fixed distance away: the width of the map.
 			// This makes the production timing independent of spawnpoint
-			var dropPos = self.Location + exit.ExitCell;
-			var startPos = dropPos + new CVec(owner.World.Map.Bounds.Width, 0);
-			var endPos = new CPos(owner.World.Map.Bounds.Left - 5, dropPos.Y);
+			var startPos = self.Location + new CVec(owner.World.Map.Bounds.Width, 0);
```

Comment:
:-1: to making this ignore the exit (which is a standard part of the production code) and always targeting the top-left cell of the actor.  You should be able to resolve the exit location when you need it, instead.